OpenRA.Game/WRot.cs

```diff
@@ -29,6 +29,27 @@ public struct WRot : IEquatable<WRot>
 		public static WRot operator -(WRot a, WRot b) { return new WRot(a.Roll - b.Roll, a.Pitch - b.Pitch, a.Yaw - b.Yaw); }
 		public static WRot operator -(WRot a) { return new WRot(-a.Roll, -a.Pitch, -a.Yaw); }
 
+		public static WRot operator *(WRot a, WRot b)
+		{
+			if (a == Zero)
+				return b;
```

Comment:
This sounds really odd when you think about it, which makes me think that calling this `Zero` was a mistake. But not sure what to do about it now.

Reply:
Good point. Any objection to `WRot.None`? `WRot.Identity` is a bit less obvious.

There are hardly any uses, so I can scope creep this PR to include a rename.

Reply:
`WRot.None` sounds good to me. :+1: